protocol/v2/types/domain.go

```diff
@@ -2,10 +2,13 @@ package types
 
 import (
 	spectypes "github.com/bloxapp/ssv-spec/types"
+
+	"github.com/bloxapp/ssv/networkconfig"
 )
 
+// TODO: get rid of singleton, pass domain as a parameter
```

Comment:
@nkryuchkov @Tomblox1 please create a ticket

Reply:
It's left like that because this change would require a lot of modifications, which would increase the size of the PR, increase the code review complexity, and delay merging of the PR. Also it looks like something that could be extracted and done separately